tests/Nox.Types.Tests/Types/NoxUri/NoxUriTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Security.Policy;
+using FluentAssertions;
+using Xunit.Abstractions;
+
+namespace Nox.Types.Tests.Types;
+
+public class UriTests
+{
+
+    private const string SAMPLE_URI = "https://user:password@www.contoso.com:80/Home/Index.htm?q1=v1&q2=v2#FragmentName";
+    private readonly ITestOutputHelper _testOutputHelper;
+
+
+    public UriTests(ITestOutputHelper testOutputHelper)
+    {
+        _testOutputHelper = testOutputHelper;
+    }
+
+    [Fact]
+    public void NoxUri_ShouldBe_SameAs_SystemUri()
```

Comment:
Can you remove the Nox Prefix?

Reply:
I think this can stay, used in Test Method name to differentiate between System.Uri